experimental/traffic-portal/src/app/core/cache-groups/divisions/table/divisions-table.component.ts

```diff
@@ -121,15 +121,16 @@ export class DivisionsTableComponent implements OnInit {
 				});
 				ref.afterClosed().subscribe(result => {
 					if(result) {
-						this.api.deleteDivision(data.id);
+						this.api.deleteDivision(data.id).then(async () => this.divisions = this.api.getDivisions());
 					}
 				});
 				break;
 			case "edit":
 				await this.router.navigate(["/core/division", data.id]);
 				break;
 			case "viewRegions":
-				console.log("Regions not implemented");
+				await this.router.navigate(["/core/regions"]);
```

Comment:
This is supposed to be a link to the regions *of the selected division(s)*, not just all regions. If you're going to implement this, you need to navigate to a filtered table view.

Reply:
It should also be a link like below, but I suspect you'd rather just not implement that.